src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import {
+  ClassBasedComponent,
+  CreateElementComponent,
+  FunctionalComponent,
+  PureComponent,
+} from "./components/index";
+
+const app = (
+  <React.Fragment>
```

Comment:
In general, everything is correct, but I'd prefer to use <> </> instead of writing React.Fragment, both of the identical.